source/audit-logs/activating-audit-logs.rst

```diff
@@ -0,0 +1,134 @@
+###############################
+Log creation and access
+###############################
+
+.. warning::
+
+    The following service is still in a technical preview and as such, will have
+    limitations on features and scope. Any feedback is appreciated and can be
```

Comment:
Q: Do we want to list the services and/or API actions that are supported in Audit Log, and add it to over time?

Reply:
I think that would be a good idea yeah. List the limitations so customers know what to expect.